### PR TITLE
changed URL to default apprentice page

### PR DESCRIPTION
**Pull request by aaronclarkcodes**
Insert description of PR here

---

## ✅️ By submitting this PR, I have verified the following

- [] Checked to see if a similar PR has already been opened 🤔️
- [] Reviewed the contributing guidelines 🔍️
- [] Added my name to the bottom of the list under the **Credits** section in the `README.md` with a link to my website or GitHub profile 👥️

### CODE REVIEW

File: content/apprenticeships/techtonic.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 company: "Techtonic"
 description: "Lots of schools teach code. At Techtonic, we empower our apprentices with hands-on development of real client projects to learn how to apply their knowledge to solve business problems."
 image: "techtonic.jpg"
-link: "https://www.techtonic.com/"
+link: "https://www.techtonic.com/apprenticeships/overview/"
```
Comment (benjaminmodayil) [inline content/apprenticeships/techtonic.md]: Hey @aaronclarkcodes this location leads to a 404 page. Seems the ending slash is the problem. If you remove it, the link works properly.

File: content/apprenticeships/barclays.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+---
+company: "Barclays"
+description: "Kickstart your career in Technology with a Foundation or Higher Apprenticeship."
+image: "barclays.jpg"
+link: "https://joinus.barclays/eme/apprenticeships/"
+location:
+  - "London, Northampton or Radbroke Hall in Knutsford"
```
Comment (benjaminmodayil) [inline content/apprenticeships/barclays.md]: For parsing reasons, can you reformat this like the other files?
```md
location:
  - "San Francisco, CA"
  - "Salt Lake City, UT"
  - "Austin, TX"
```

File: content/apprenticeships/techtonic.md (hunk 1 above)
Comment (aaronclarkcodes) [reply content/apprenticeships/techtonic.md]: @benjaminmodayil changed

File: content/apprenticeships/barclays.md (hunk 2 above)
Comment (aaronclarkcodes) [reply content/apprenticeships/barclays.md]: @benjaminmodayil changed

**Comment by benjaminmodayil** [comment]
Hi @aaronclarkcodes I noticed you added "barclays.jpg" to the markdown file. Did you add an actual image to the repository under static/images/apprenticeships? I didn't see it when I previewed the changes. Can you right-click + save the screenshot below to the correct folder? Just note, that this is a PNG, you should be okay to change the file ending to a `jpg` to match your markdown file.

barclays

**Comment by aaronclarkcodes** [comment]
Yes I can do that.

Curious as to why you don't make the small changes vs commenting. Seems inefficient.

**Pull request merged**